htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -728,20 +728,20 @@ function print_left_eldy_menu($db, $menu_array_before, $menu_array_after, &$tabM
 
 	if (is_array($moredata) && !empty($moredata['searchform'])) {	// searchform can contains select2 code or link to show old search form or link to switch on search page
 		print "\n";
-		print "<!-- Begin SearchForm -->\n";
-		print '<div id="blockvmenusearch" class="blockvmenusearch">'."\n";
+		//print "<!-- Begin SearchForm -->\n";
```

Comment:
What is goal of removing this line ?

Reply:
It’s just a suggestion for clean html in production.
[Idea] > Maybe comments should only be visible in debug mod (if debug : display comments in dom; else: none;) ?

Reply:
The goal of such comment is to be able when viewed in production (after a bug report for example) to quickly find the code on development sources.
They are on purpose on production to make bug fixes and impact analysis from a browser very fast.